scico/loss.py

```diff
@@ -154,7 +161,7 @@ def prox(self, x: Union[JaxArray, BlockArray], lam: float) -> Union[JaxArray, Bl
     @property
     def hessian(self) -> linop.LinearOperator:
         r"""If ``self.A`` is a :class:`.LinearOperator`, returns a new :class:`.LinearOperator` corresponding
-        to Hessian :math:`\mathrm{A^*A}`.
+        to Hessian :math:`2 \mathrm{scale} \cdot \mathrm{A^* A}`.
 
```

Comment:
Docs were wrong in omission of 2 * scale? Also, not sure the `\cdot` is notationally necessary for scalar multiplication.

Reply:
Yes, they were incorrect.

I added the `\cdot` to keep things consistent with other parts of the repo. I agree though, we could remove all of the `\cdot`s where  the are unnecessary.

Reply:
Perhaps you could remove them as part of this PR?